src/JuMP.jl

```diff
@@ -619,15 +613,6 @@ Returns number of variables in `model`.
 """
 num_variables(model::Model)::Int64 = MOI.get(model, MOI.NumberOfVariables())
 
-"""
-    num_nonlinear_constraints(model::Model)
-
-Returns the number of nonlinear constraints associated with the `model`.
```

Comment:
Is this method being deleted or moved somewhere else? I couldn't find easily find the new version if it moved.

Reply:
No public facing functions were deleted or changed. It's now here:
https://github.com/jump-dev/JuMP.jl/blob/bbd8d99a6cc639d928956994d70e4c3343d192bd/src/nlp.jl#L429-L436

The diff is a little hard to navigate. And it made sense to move this to the nonlinear code, rather than have it scattered in JuMP.jl.